nodes/list_mutators/vd_attr_node_mk2.py

```diff
@@ -0,0 +1,242 @@
+# This file is part of project Sverchok. It's copyrighted by the contributors
+# recorded in the version control history of the file, available from
+# its original location https://github.com/nortikin/sverchok/commit/master
+#
+# SPDX-License-Identifier: GPL3
+# License-Filename: LICENSE
+
+import json
+import copy
+
+import bpy
+from bpy.props import BoolProperty, StringProperty, IntProperty, CollectionProperty
+from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.data_structure import updateNode, enum_item_5
+from sverchok.nodes.viz.viewer_3d import SvViewer3D
+
+
+sock_str = {
+    'enum': "SvStringsSocket",
+    '3f': "SvVerticesSocket",
+    '4f': "SvColorSocket",
+    'i': "SvStringsSocket",
+    'b': "SvStringsSocket"
+}
+
+
+socket_colors = {
+    "SvStringsSocket": (0.6, 1.0, 0.6, 1.0),
+    "SvVerticesSocket": (0.9, 0.6, 0.2, 1.0),
+    "SvColorSocket": (0.9, 0.8, 0.0, 1.0),
+}
+
+
+def props(**x):
+    prop = lambda: None
+    _ = [setattr(prop, k, v) for k, v in x.items()]
+    return prop
+
+# this dict (ordered by default as per python 3.7 ?..) determines the order of sockets
+maximum_spec_vd_dict = dict(
+    activate=props(name="activate", kind="b"),
+    display_verts=props(name="display verts", kind="b"),
+    display_edges=props(name="display edges", kind="b"),
```

Comment:
Why not to use namedtuple?

Reply:
I was not thinking about changing the logic of this node (just adapting it to the new node) but i will look into it

Reply:
By the moment i'm going to leave it as it was, maybe a time comes for the VD attribute to be refactored...